### PR TITLE
Feature Login Front End #158593811

### PR DESCRIPTION
**Pull request by ghost**
**What does this PR do?**
Connects login page to the API endpoint `POST api/v1/auth/login`

**Description of tasks to be completed**
- Enable login
- Store token somewhere accessible for future requests
- Redirect to user's dashboard if the role is "Consumer"
- Redirect to admin's dashboard if the role is "Administrator"

**What are the relevant Pivotal Tracker stories?**
[#158593811](https://www.pivotaltracker.com/story/show/158593811)

### CODE REVIEW

File: login/login.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    event.preventDefault();
+    login(email.value, password.value);
+}
+function login(email, password) {
+    // endpoint to be used
+    let endpoint = '/auth/login'
+
+
+    // the request body format
+    let requestBody = {
```
Comment (denniswanjiru) [inline login/login.js:20]: If key and value use the same name you can do something like this. 
```
let requestBody = { email, password}

```
It works the same as the code you've written though this is shorter and cleaner

Comment (ghost) [reply login/login.js:20]: Noted...

**Comment by denniswanjiru** [comment]
You can always consider separating your static HTML files from js files and you should ignore pushing the bundle.js if it's generated by webpack

**Comment by ghost** [comment]
Well, if I don't push the bundle.js, I get errors when my code runs in the browser... stuff like "cannot resolve 'require' "

**Comment by denniswanjiru** [comment]
Are you using github pages?

**Pull request merged**